src/external/druidExternal.ts

```diff
@@ -1310,6 +1315,14 @@ module Plywood {
         return DruidExternal.timePartToExtraction(action.part, action.getTimezone());
       }
 
+      if (action instanceof TransformCaseAction) {
+        var transformType = DruidExternal.caseToDruid[action.transformType];
+        if (!transformType) throw new Error(`unsupported case transformation '${transformType}': supported transformations are: ${Object.keys(DruidExternal.caseToDruid)}`);
```

Comment:
This will log `"unsupported case transformation 'blah': supported transformations are: [object Array]"`I think it is ok for it to be less descriptive here, we do not really expect this error to happen outside of test